diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-0.5.1/createlinks mezzanine_patched_e-smith-dnscache-0.5.1/createlinks
--- e-smith-dnscache-0.5.1/createlinks	2005-09-14 19:39:10.862034544 -0600
+++ mezzanine_patched_e-smith-dnscache-0.5.1/createlinks	2005-09-14 19:38:24.892023048 -0600
@@ -2,11 +2,6 @@
 
 use esmith::Build::CreateLinks  qw(:all);
 
-templates2events('/var/service/dnscache/root/servers/@',
-    qw(bootstrap-console-save console-save));
-templates2events('/var/service/dnscache.forwarder/config',
-    qw(post-install post-upgrade));
-
 foreach my $event (qw(
 	console-save
 	network-create
@@ -18,6 +13,7 @@
 	))
 {
     safe_symlink("restart", "root/etc/e-smith/events/$event/services2adjust/dnscache");
+    safe_symlink("restart", "root/etc/e-smith/events/$event/services2adjust/dnscache.forwarder");
 }
 
 event_link("initialize-default-databases", "dns-update", "00");
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-0.5.1/root/var/service/dnscache/run mezzanine_patched_e-smith-dnscache-0.5.1/root/var/service/dnscache/run
--- e-smith-dnscache-0.5.1/root/var/service/dnscache/run	2005-09-14 19:39:10.863034392 -0600
+++ mezzanine_patched_e-smith-dnscache-0.5.1/root/var/service/dnscache/run	2005-09-14 19:37:34.103744040 -0600
@@ -27,10 +27,13 @@
 use esmith::DomainsDB;
 use esmith::NetworksDB;
 use esmith::util;
+use esmith::templates;
 
 sub allow_networks_2access_cache;
 sub delegate_domains_2DNS;
 
+processTemplate({ TEMPLATE_PATH => "/var/service/dnscache/root/servers/@" });
+
 unless (-f "/var/service/dnscache/seed")
 {
     system(qw(/bin/dd
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-0.5.1/root/var/service/dnscache.forwarder/run mezzanine_patched_e-smith-dnscache-0.5.1/root/var/service/dnscache.forwarder/run
--- e-smith-dnscache-0.5.1/root/var/service/dnscache.forwarder/run	2005-05-13 15:45:16.000000000 -0600
+++ mezzanine_patched_e-smith-dnscache-0.5.1/root/var/service/dnscache.forwarder/run	2005-09-14 19:33:54.698098768 -0600
@@ -1,4 +1,10 @@
 #! /bin/sh
 exec 2>&1
+
+for file in config root/servers/@
+do
+    /sbin/e-smith/expand-template /var/service/dnscache.forwarder/$file
+done
+
 . ./config
 exec envuidgid dnscache softlimit -o250 -d "$DATALIMIT" /usr/local/bin/dnscache
