diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run
--- e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2008-08-08 16:48:07.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2008-08-08 16:47:43.000000000 -0400
@@ -1,6 +1,12 @@
 #! /bin/sh
 exec 2>&1
 . ./config
+
+# Ensure that dnscache has source of randomness to initialise its source
+# port RNG
+test -f ./seed ||  /bin/dd if=/dev/urandom bs=128 count=1 of=seed
+exec<./seed
+
 # http://marc.theaimsgroup.com/?l=djbdns&m=104812086607532&w=2
 trap "" SIGPIPE
 exec envuidgid dnscache softlimit -o250 -d "$DATALIMIT" /usr/local/bin/dnscache
