diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache/run mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/run
--- e-smith-dnscache-1.0.0/root/var/service/dnscache/run	2007-09-06 13:01:41.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/run	2007-09-06 13:01:10.000000000 -0400
@@ -117,6 +117,8 @@
 $ENV{CACHESIZE} = 200000;
 $ENV{ROOT} = '/service/dnscache/root';
 
+# http://marc.theaimsgroup.com/?l=djbdns&m=104812086607532&w=2
+$SIG{'PIPE'} = 'IGNORE';
 exec("envuidgid", "dnscache", "softlimit", "-o250", "-d", "$datalimit",
     "/usr/local/bin/dnscache")
     or die "failed to execute envuidgid/softlimit/dnscache: $!";
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run
--- e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2005-05-13 17:45:16.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2007-09-06 13:01:23.000000000 -0400
@@ -1,4 +1,6 @@
 #! /bin/sh
 exec 2>&1
 . ./config
+# http://marc.theaimsgroup.com/?l=djbdns&m=104812086607532&w=2
+trap "" SIGPIPE
 exec envuidgid dnscache softlimit -o250 -d "$DATALIMIT" /usr/local/bin/dnscache
