diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/root/servers/@ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/root/servers/@
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/root/servers/@	2006-03-14 15:59:06.000000000 -0500
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/root/servers/@	2008-06-18 15:18:55.000000000 -0400
@@ -11,7 +11,7 @@
 			192.36.148.17
 			192.58.128.30
 			193.0.14.129
-			198.32.64.12
+			199.7.83.42
 			202.12.27.33
 		    );
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/root/servers/@ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/root/servers/@
--- e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/root/servers/@	2005-06-06 14:48:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/root/servers/@	1969-12-31 19:00:00.000000000 -0500
@@ -1,13 +0,0 @@
-198.41.0.4
-192.228.79.201
-192.33.4.12
-128.8.10.90
-192.203.230.10
-192.5.5.241
-192.112.36.4
-128.63.2.53
-192.36.148.17
-192.58.128.30
-193.0.14.129
-198.32.64.12
-202.12.27.33
