diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE	2005-04-20 17:58:58.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE	2007-10-16 21:51:28.000000000 -0400
@@ -1,4 +1,4 @@
 {
-    my $c = $dnscache{'CacheSize'} || 1000000;
+    my $c = $dnscache{'CacheSize'} || 10000000;
     "export CACHESIZE=$c"
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT	2005-04-20 14:32:37.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT	2007-10-16 21:52:37.000000000 -0400
@@ -1,5 +1,4 @@
 {
-    my $datalimit = $dnscache{'DataLimit'} || 3000000;
+    my $datalimit = $dnscache{'DataLimit'} || 12000000;
     "DATALIMIT=$datalimit";
 }
-
