diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/CACHESIZE	2005-04-20 17:58:58.000000000 -0400
@@ -1,4 +1,4 @@
 {
-    my $c = $dnscache{'CacheSize'} || 10000000;
+    my $c = $dnscache{'CacheSize'} || 1000000;
     "export CACHESIZE=$c"
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/DATALIMIT	2005-04-20 14:32:37.000000000 -0400
@@ -1,4 +1,5 @@
 {
-    my $datalimit = $dnscache{'DataLimit'} || 12000000;
+    my $datalimit = $dnscache{'DataLimit'} || 3000000;
     "DATALIMIT=$datalimit";
 }
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/FORWARDER mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/FORWARDER
--- e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/FORWARDER	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/etc/e-smith/templates/var/service/dnscache.forwarder/config/FORWARDER	2005-04-20 14:32:37.000000000 -0400
@@ -1,4 +1,4 @@
 {
-    (defined $dnscache{'Forwarder'} && $dnscache{'Forwarder'}) ?
+    (defined $dnscache{'Forwarder'}) ?
 	"export FORWARDONLY=1" : "";
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/@ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/@
--- e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/@	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/@	2005-06-06 14:48:01.000000000 -0400
@@ -1 +1,13 @@
-127.0.0.2
+198.41.0.4
+192.228.79.201
+192.33.4.12
+128.8.10.90
+192.203.230.10
+192.5.5.241
+192.112.36.4
+128.63.2.53
+192.36.148.17
+192.58.128.30
+193.0.14.129
+198.32.64.12
+202.12.27.33
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/127.in-addr.arpa mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/127.in-addr.arpa
--- e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/127.in-addr.arpa	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/root/servers/127.in-addr.arpa	2003-01-30 17:34:15.000000000 -0500
@@ -0,0 +1 @@
+127.0.0.1
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache/run mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/run
--- e-smith-dnscache-1.0.0/root/var/service/dnscache/run	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache/run	2008-06-18 14:48:50.000000000 -0400
@@ -92,13 +92,18 @@
 	$nameserver = undef;
     }
     $nameserver ||= ($systemlocalnetwork eq 'yes') ? $tinydns_ip : 'none';
-    push @localnetworks,
-        esmith::util::computeAllLocalNetworkPrefixes ($key, $mask);
+    my @all_network_prefixes = esmith::util::computeAllLocalNetworkPrefixes ($key, $mask);
+    push @localnetworks, @all_network_prefixes;
 
-    my $reverse = esmith::util::computeLocalNetworkReversed ($key, $mask);
-    # Remove the trailing period provided by computeLocalNetworkReversed
-    $reverse =~ s/\.$//;
-    $reversenets{$reverse} = $nameserver unless ($nameserver eq 'none');
+    unless ($nameserver eq 'none')
+    {
+	foreach (@all_network_prefixes)
+	{
+	    my @quads = split(/\./, $_);
+	    my $reverse_zone = join('.', reverse @quads) . ".in-addr.arpa";
+	    $reversenets{$reverse_zone} = $nameserver;
+	}
+    }
 }
 
 allow_networks_2access_cache(@localnetworks);
@@ -117,8 +122,6 @@
 $ENV{CACHESIZE} = 200000;
 $ENV{ROOT} = '/service/dnscache/root';
 
-# http://marc.theaimsgroup.com/?l=djbdns&m=104812086607532&w=2
-$SIG{'PIPE'} = 'IGNORE';
 exec("envuidgid", "dnscache", "softlimit", "-o250", "-d", "$datalimit",
     "/usr/local/bin/dnscache")
     or die "failed to execute envuidgid/softlimit/dnscache: $!";
diff -Nur -x '*.orig' -x '*.rej' e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run
--- e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2008-06-18 14:53:01.000000000 -0400
+++ mezzanine_patched_e-smith-dnscache-1.0.0/root/var/service/dnscache.forwarder/run	2005-05-13 17:45:16.000000000 -0400
@@ -1,6 +1,4 @@
 #! /bin/sh
 exec 2>&1
 . ./config
-# http://marc.theaimsgroup.com/?l=djbdns&m=104812086607532&w=2
-trap "" SIGPIPE
 exec envuidgid dnscache softlimit -o250 -d "$DATALIMIT" /usr/local/bin/dnscache
